examples/Cassie/dispatcher_robot_out.cc

```diff
@@ -91,15 +110,27 @@ int do_main(int argc, char* argv[]) {
 
   auto diagram = builder.Build();
 
-  systems::UDPDrivenLoop loop(*diagram, *input_sub, nullptr);
+  if (FLAGS_simulation) {
+    drake::systems::lcm::LcmDrivenLoop loop(*diagram, *sim_input_sub, nullptr,
+        &lcm_local,
+        std::make_unique<UtimeMessageToSeconds<dairlib::lcmt_cassie_out>>());
+
+    // caused an extra publish call?
+    loop.set_publish_on_every_received_message(true);
```

Comment:
I forgot what you mentioned last time about the publish issue here. I think lcm driven loop forces LcmPublisherSystem to publish when a message arrives in LcmSubscriberSystem. When I set `set_publish_on_every_received_message` to false, the publish rate is 1k hz as expected. And when I set `set_publish_on_every_received_message` to true, the publish rate increased to 2k hz.

Reply:
I was just wondering if the problem causes us trouble. Maybe it's a bug in drake, but it doesn't matter for the users like us, as long as it publishes at the rate we want.

Reply:
Good catch--the upstream issue has actually been fixed. The problem here was more mundane--both the simulator and dispatcher were publishing. Fixed with latest commit.